### DISCUSSION TITLE
curvi / CANopen

### DISCUSSION THREAD

**Discussion by dssatech** [community/community#31686] [Packages]
Hello
I am trying to use curvi/CANopen package, and can not get it to build...
How do I setup the folders to include the supplied *.c *.h files ?
arduino build says it cannot find them...

**Comment by jfullstackdev** [comment]
maybe it's pointing to a wrong directory, make sure you follow that directory structure too

and if this is the same project we are referring to, you can raise an issue there and ask the question

https://github.com/curvi/CANopen/issues
